packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3324,7 +3318,7 @@ export class ContainerRuntime
 						fetchedSnapshotRefSeq: fetchResult.latestSnapshotRefSeq,
 					},
 				);
-				this.closeFn(error);
+				this.disposeFn(error);
```

Comment:
@markfields As a note, the "ContainerClose" error event will no longer happen because of this. But there are relevant error handlers around the summarizer code that still log the errors.

Reply:
Good point.  I'm putting a PR out to log close on dispose if we're not already closed.

Reply:
#15938 - Up for debate though

Reply:
I closed that one and opened some ADO items for Kian to follow up on.